doc/xml/reference.xml

```diff
@@ -1096,6 +1132,15 @@
                         <example>backupdb</example>
                     </config-key>
 
+                    <!-- ======================================================================================================= -->
+                    <config-key id="pg-local" name="PostgreSQL Locality">
+                        <summary><postgres/> is local.</summary>
+
+                        <text>Indicates that <postgres/> for the specified index is local and any <postgres/> host settings (e.g. <setting>pg1-host</setting>) should be ignored.</text>
```

Comment:
Same comment as for repo-local.

Reply:
Fixed in commit 2023430cbc97bb5deaf0f28372ecaebc1bbce3f7

Reply:
Looks good to me.